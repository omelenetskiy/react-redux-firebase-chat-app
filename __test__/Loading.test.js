import React from 'react';
import Loading from '../src/components/Loading';
import { shallow } from 'enzyme';

it('should render a Loading', () => {
	const wrapper = shallow(<Loading />);
	expect(wrapper).toMatchSnapshot();
});
