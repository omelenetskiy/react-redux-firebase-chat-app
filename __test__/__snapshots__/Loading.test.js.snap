// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Loading 1`] = `
<div>
  Loading...
</div>
`;
